cvat/apps/dataset_manager/formats/yolo.py

```diff
@@ -44,11 +59,117 @@ def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8_oriented_boxes', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@exporter(name='YOLOv8 Segmentation', ext='ZIP', version='1.0')
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8_segmentation', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@exporter(name='YOLOv8 Pose', ext='ZIP', version='1.0')
+def _export(dst_file, temp_dir, instance_data, save_images=False):
+    with GetCVATDataExtractor(instance_data, include_images=save_images) as extractor:
+        dataset = Dataset.from_extractors(extractor, env=dm_env)
+        dataset.export(temp_dir, 'yolov8_pose', save_images=save_images)
+    make_zip_archive(temp_dir, dst_file)
+
+
+@importer(name='YOLOv8 Detection', ext="ZIP", version="1.0")
+def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+    _import_common(
+        src_file,
+        temp_dir,
+        instance_data,
+        load_data_callback=load_data_callback,
+        format_name="yolov8",
+        **kwargs
+    )
+
+
+@importer(name='YOLOv8 Segmentation', ext="ZIP", version="1.0")
+def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+    _import_common(
+        src_file,
+        temp_dir,
+        instance_data,
+        load_data_callback=load_data_callback,
+        format_name="yolov8_segmentation",
+        **kwargs
+    )
+
+
+@importer(name='YOLOv8 Oriented Bounding Boxes', ext="ZIP", version="1.0")
+def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+    _import_common(
+        src_file,
+        temp_dir,
+        instance_data,
+        load_data_callback=load_data_callback,
+        format_name="yolov8_oriented_boxes",
+        **kwargs
+    )
+
+
+@importer(name='YOLOv8 Pose', ext="ZIP", version="1.0")
+def _import(src_file, temp_dir, instance_data, load_data_callback=None, **kwargs):
+    if TaskData.META_FIELD in instance_data.meta:
+        labels = instance_data.meta[TaskData.META_FIELD]["labels"]
+    else:
+        labels = instance_data.meta[ProjectData.META_FIELD]["labels"]
+
+    true_skeleton_point_labels = {
+        skeleton_label["name"]: [
+            el.attrib["data-label-name"]
+            for el in ET.fromstring("<root>" + skeleton_label.get("svg", "") + "</root>")
+            if el.tag == "circle"
+        ]
+        for _, skeleton_label in labels
+        if skeleton_label["type"] == str(LabelType.SKELETON)
+    }
```

Comment:
Shouldn't there be some equivalent code in the corresponding export function? Otherwise, how will Datumaro know what order to put the points in?

Reply:
datumaro can read the order from the dataset. PointsCategories contains ordered point labels for each skeleton

Reply:
Ah, I see now. In that case, would you mind factoring out a helper function that does the equivalent of this:

https://github.com/cvat-ai/cvat/blob/a9d250df640a546a928ae5cacdb11b3ae64f86cc/cvat/apps/dataset_manager/bindings.py#L1459-L1465

so that import and export can reuse the same logic?

Reply:
reused the same logic which is used on export